cmd/put.go

```diff
@@ -103,7 +117,9 @@ func init() {
 	// Cobra supports local flags which will only run when this command
 	// is called directly, e.g.:
 	putCmd.Flags().StringP("kms-key-id", "k", "", "KMS CMK ID to use for encryption")
+	putCmd.Flags().BoolP("verbose-output", "V", false, "Set to output the version id of the uploaded object")
 	putCmd.MarkFlagRequired("kms-key-id")
 	viper.BindPFlag("kmskeyid", putCmd.Flags().Lookup("kms-key-id"))
+	viper.BindPFlag("verboseoutput", putCmd.Flags().Lookup("output-version-id"))
```

Comment:
Does this still work? Doesn't seem like it would since you changed the name to `verbose-output`